code/modules/recycling/disposal.dm

```diff
@@ -1052,15 +1054,34 @@
 
 	if(istype(I, /obj/item/destTagger))
 		var/obj/item/destTagger/O = I
+		var/tag = uppertext(GLOB.TAGGERLOCATIONS[O.currTag])
+		playsound(src.loc, 'sound/machines/twobeep.ogg', 100, 1)
+		if(O.currTag in sort_type)
+			sort_type -= O.currTag
+			to_chat(user, "<span class='notice'>Removed [tag] from filter.</span>")
+		else
+			sort_type += O.currTag
```

Comment:
hmm dumb question, but would this be better as `|=`?
```suggestion
			sort_type |= O.currTag
```

Reply:
Works either way